moveit/package.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<package>
+  <name>moveit</name>
+  <version>0.0.0</version>
+  <description>Meta package that contains all essential package of MoveIt!. Until Summer 2016 MoveIt! had been developed over multiple repositories, where developers' usability and maintenance effort was non-trivial. See <a href = "http://discourse.ros.org/t/migration-to-one-github-repo-for-moveit/266/34">the detailed discussion for the merge of several repositories</a>.</description>
+  <maintainer email="David.T.Coleman@colorado.edu">Dave Coleman</maintainer>
+  <maintainer email="imcmahon01@gmail.com">Ian McMahon</maintainer>
+  <maintainer email="130s@2000.jukuin.keio.ac.jp">Isaac I. Y. Saito</maintainer>
+  <maintainer email="mfergs7@gmail.com">Michael Ferguson</maintainer>
+  <maintainer email="goerner@informatik.uni-hamburg.de">Michael Görner</maintainer>
```

Comment:
please change that to me@v4hn.de . This is the address I decided to use in all commits.